.eslintrc.json

```diff
@@ -12,6 +12,7 @@
       "line": { "exceptions": ["/"], "markers": ["/"] },
       "block": { "exceptions": ["*"], "markers": ["/"], "balanced": true }
     }],
-    "space-before-function-paren": ["error", { "anonymous": "never", "named": "never", "asyncArrow": "always" }]
+    "space-before-function-paren": ["error", { "anonymous": "never", "named": "never", "asyncArrow": "always" }],
+    "react/react-in-jsx-scope": 0
```

Comment:
" Configuration for rule "react/react-in-jsx-scope" is invalid: Severity should be one of the following: 0 = off, 1 = warn, 2 = error (you passed 'false')."
Maybe i slould use off here?